config/routes.rb

```diff
@@ -14,6 +14,7 @@
 
   get '/content', to: 'content#show'
   get '/single_page/*base_path', to: 'single_item#show', defaults: { format: :json }
+  get '/single_page', to: 'single_item#show', defaults: { base_path: '', format: :json }
```

Comment:
What are the downsides to setting a default base_path value of "/"?

Reply:
The code will pre-pend the leading `/` before querying by :base_path